### ISSUE TITLE
Linux install instructions

### ISSUE THREAD

**Issue by duramato** [Stephen304/bitcannon#83]
Hello, are there any docs on how to install on linux i can only find guides/docs regarding windows..

**Comment by Stephen304** [comment]
Linux is much easier than the other platforms, but there are instructions on the wiki anyway: https://github.com/Stephen304/bitcannon/wiki#installing-bitcannon

**Comment by duramato** [comment]
@Stephen304 Thanks! It was preety straight forword to run ,the only hard part was getting it in to my server (maybe switch mega.co.nz by another hoster? ) ,also i added a sample upstart script to someone who might want it ,as i need to mke my own :P (https://github.com/Stephen304/bitcannon/wiki/Installing-BitCannon)
One more thing is it possible to add more website dumps in the config?

**Comment by OmgImAlexis** [comment]
@duramato yeah, any you add to the config will get polled when it does it's update. If you want kat I'd suggest you apply for the API otherwise you'll keep getting failures because they changed their API endpoints.

**Comment by spice-is-nice** [comment]
Hello, in my CentOS server I created the following /etc/init/bitcannon.conf but bitcannon doesn't automatically start on reboot.
```
setuid root
setgid root

start on runlevel [2345]

stop on runlevel [016]
chdir /opt/bitcannon
exec /opt/bitcannon/bitcannon_linux_386

```
/var/log/messages shows:
`/etc/init/bitcannon.conf:1: Unknown stanza
`
when I remove 
setuid root
setgid root
and reboot bitcannon still doesn't start and there are no errors in messages. Bitcannon runs fine when I start it from the command line.
Any suggestions?
